### PR TITLE
Move to semantic ui and render the reservation (wait time) after form submission

### LINKED ISSUE #20: Home page, including form to enter caller details
Mock: https://gallery.googleplex.com/projects/MCHbtQVoQ2HCZR4FoCy-jZSS/files/MCEJu8Y2hyDScVh4FS5EAYuwSeJISu4nZDA

- Calls "GET /queues" to retrieve list of queues
- Calls "POST /reservations" to save reservation

### PR DESCRIPTION
**Pull request by mscharfstein**
Fixes #20, #21 

The new pages look like this:
https://user-images.githubusercontent.com/29430896/80753281-00056780-8afb-11ea-97c1-02a7bdfa7d1a.png
https://user-images.githubusercontent.com/29430896/80753287-03005800-8afb-11ea-8bed-ad6e33b4010a.png
(these are zoomed in, they don't take up the whole page they are just centered in the middle of the page)
Going to rely on some of Marc's expertise to make these things look better :)

- [x] Tests pass
- [x] Appropriate changes to README are included in PR

### CODE REVIEW

File: callmeback/frontend/src/components/Home.jsx [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
-  const classes = useStyles();
+function Home(props) {
   const [name, setName] = useState('')
   const [number, setNumber] = useState('')
   const [topic, setTopic] = useState('')
   const [requestButtonEnabled, setRequestButtonEnabled] = useState(false)
+  const history = useHistory();
 
   useEffect(() => {
     if (number !== "") {
```
Comment (marcja) [inline callmeback/frontend/src/components/Home.jsx:13]: Use regular expression? For example:
"""
const rePhoneNumber = /((\(\d{3}\)?)|(\d{3}))([\s-./]?)(\d{3})([\s-./]?)(\d{4})/;
...
if (number.match(rePhoneNumber) {
"""

File: callmeback/frontend/src/components/Home.jsx [hunk 2]
```diff
@@ -20,84 +17,63 @@ function Home() {
...
+        <br/>
+        <Form onSubmit={handleSubmit}>
+        <Form.Field 
+          control={Input}
+          autoComplete
+          value={name}
+          placeholder='Name (optional)'
+          onChange={(evt) => {setName(evt.target.value)}}
+        />
+        <Form.Field
```
Comment (marcja) [inline callmeback/frontend/src/components/Home.jsx:53]: Add HTML5 type for telephone numbers:
```
type="tel"
pattern="((\(\d{3}\)?)|(\d{3}))([\s-./]?)(\d{3})([\s-./]?)(\d{4})"
```

File: callmeback/frontend/src/components/Home.jsx [hunk 3]
```diff
@@ -20,84 +17,63 @@ function Home() {
...
+          required 
+          autoComplete
+          control={Input} 
+          value={number}
+          placeholder='Phone number' 
+          onChange={(evt) => {setNumber(evt.target.value)}} 
+        />
+        <Form.TextArea
+          placeholder="What can we help you with?"
```
Comment (marcja) [inline callmeback/frontend/src/components/Home.jsx:62]: Also "(optional)"?

File: callmeback/frontend/src/components/Reservation.jsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+function Reservation() {
+    const { id } = useParams();
+    const [reservationDetails] = useState(() => {
+        // TODO: Call reservations/{id} endpoint to get reservation data
+        // (this would not be necessary if we had a global redux state with the
+        // reservation data we got upon form submit)
+        // and generate a random expected wait time here.
+        const topic = "Employment";
+        const expCallStart = new Date();
```
Comment (marcja) [inline callmeback/frontend/src/components/Reservation.jsx:14]: Look into adding [moment.js](https://momentjs.com/) as a dependency. You will find it very useful.

File: callmeback/frontend/src/components/Home.jsx (hunk 1 above)
Comment (mscharfstein) [reply callmeback/frontend/src/components/Home.jsx:13]: good idea!

File: callmeback/frontend/src/components/Reservation.jsx (hunk 4 above)
Comment (mscharfstein) [reply callmeback/frontend/src/components/Reservation.jsx:14]: Cool, i'll do that in a separate PR

**Pull request merged**